model_training/tests/data_preparation/test_data_preprocess.py

```diff
@@ -34,12 +34,12 @@ def test_compute_layer_metrics_by_cycle(
     model_name: str, data_preprocessor: DataPreprocessor
 ) -> None:
     """Test compute_layer_metrics_by_cycle."""
-    model_dir = Path(__file__).parent / "example_raw_data" / model_name
+    model_dir = f"{BASE_DIR}/{model_name}"
```

Comment:
Can't we still use Path syntaxis here and below? 

```suggestion
    model_dir = BASE_DIR / model_name
```

Reply:
Added it in.